### DISCUSSION TITLE
Where did *beep go?

### DISCUSSION THREAD

**Discussion by cosmiccoincidence** [RE-SS3D/SS3D#717] [General]
Where in the world is our ***beep**? Where in the world can he be?

Did he get hit by a bus?
Did he fall in a well?
Did he get a job offer and move away?
Did his dog *take him* for a walk one day?

This thread will be dedicated to any information regarding what happened to our boy ***beep**.

**Comment by stilnat** [comment]
I need the full story of ***beep** , I need someone to tell me the full story like a wise grandpa would do in a cold winter evening, waiting for the snow storm to calm down.

**Comment by Dyhr** [comment]
My bet is on hit by a bus
